### DISCUSSION TITLE
How to Update Users Collection without Verified Set to True

### DISCUSSION THREAD

**Discussion by harishudewa** [pocketbase/pocketbase#2668] [Q&A]
I try to update the users collection after creating a user and I already authenticated, but the verified field still set to false.  On the pocketbase admin dashboard, the data didn't updated. But, on my terminal where I console.log the response after doing `pb.collection('users').update()`, I got the record with updated data without any errors. 

This is my API Rules for users collection,
![image](https://github.com/pocketbase/pocketbase/assets/83929596/52da2adb-edd8-4dbc-833e-55b254accbf9)

I can successfully update the record after changed verified field to true.

So, what should I do to update the data without verified field set to true?

Thanks!

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean.

Regular users are not allowed to change directly the `verified` field (_unless they have "Manage rule" access_).

The `verified` field is managened internally and will change once the user confirm their email.
If your app doesn't require users to have an email, you can safely ignore the `verified` field (at the end it is just a bool flag).
